.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Feature Request
+description: Suggest an idea for this project
+labels: ["enhancement"]
+type: Enhancement
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest improvements!
+  - type: textarea
+    id: describe-problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: |
+        A clear and concise description of what the problem is.
+        For example: I'm always frustrated when [...]
+    validations:
+      required: true
+  - type: textarea
+    id: proposed-solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: alternative-solutions
+    attributes:
+      label: What did you expect to happen?
+      description: A clear and concise description of any alternative solutions or features you've considered.
```

Comment:
The label doesn't really match the description. Maybe some more like: "Which alternative solutions did you consider?"

Reply:
Yep this description needs tweaking